test/e2e/create_test.go

```diff
@@ -154,7 +154,7 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 	// This is required for testing BYOH CSR approval feature so that BYOH instances
 	// CSR's are not approved by Cluster Machine Approver
 	expectedPodCount := int32(0)
-	err = tc.scaleMachineApproverDeployment(&expectedPodCount)
+	err = tc.scaleDeployment("openshift-cluster-machine-approver", "machine-approver", &expectedPodCount)
```

Comment:
Please add constants for these strings given the use in multiple places. Same for CVO.

Reply:
+1